src/js/dp/search.js

```diff
@@ -27,40 +27,72 @@ if (searchContainer) {
       url.searchParams.set("page", "1");
     }
     const resultsLoader = document.querySelector('#results-loading');
-    // if it takes more than 500ms to retreive results, show a loading message
-    setTimeout(() => {
-      if (resultsLoader) resultsLoader.classList.remove('hide');
-      if (scrollToTop) scrollToTopOfSearch();
-    }, 500);
 
-    const responseText = await fetchHtml(url);
+    const numOfParams = Array.from(url.searchParams).length
+
+    // Current behaviour bugs out due to search controller not rendering results if 
+    // no filters and no query is selected. This renders a message instead. 
+    const noFiltersSelected = numOfParams === 0 || numOfParams === 1 && url.searchParams.has("page");
 
```

Comment:
I believe you're wanting the last two conditions to be compared against one another? If so there needs to be brackets: `const noFiltersSelected = numOfParams === 0 || (numOfParams === 1 && url.searchParams.has("page"));`